packages/netglade_utils/lib/src/extensions/iterable_extensions.dart

```diff
@@ -0,0 +1,20 @@
+import 'dart:core';
+
+import 'package:collection/collection.dart';
+
+typedef Selector<T, K> = K Function(T selectFrom);
+
+extension IterableExtensions<T> on Iterable<T> {
+  /// Returns a set containing only distinct elements from the given collection.
```

Comment:
Od 4. slova dál to je jen definice setu, který je zmíněn na 3. slově. Potřebujem to tam tedy?

Reply:
Já to kopíroval z dokumentace Kotlinu a tam to vracelo něco jiného :D. Klidně dám pryč.